notebooks/06_Serving/06_01_Seldon_Core_Serving.ipynb

```diff
@@ -0,0 +1,185 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Create an Istio gateway in that namespace named `kubeflow-gateway`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl create -f gateway.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4. Create an example SeldonDeployment with a dummy model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl create -f seldondeployment.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5.Wait for state to become available"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl get sdep seldon-model -n testseldon -o jsonpath='{.status.state}\\n'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 6. Port forward to the Istio gateway"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl port-forward $(kubectl get pods -l istio=ingressgateway -n istio-system -o jsonpath='{.items[0].metadata.name}') -n istio-system 8004:80"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 7. Send a prediction request"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!curl -s -d '{\"data\": {\"ndarray\":[[1.0, 2.0, 5.0]]}}'    -X POST http://localhost:8004/seldon/testseldon/seldon-model/api/v1.0/predictions    -H \"Content-Type: application/json\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You should be able to see a response like below:\n",
+    "\n",
+    "```\n",
+    "{\n",
+    "  \"meta\": {\n",
+    "    \"puid\": \"i2e1i8nq3lnttadd5i14gtu11j\",\n",
+    "    \"tags\": {\n",
+    "    },\n",
+    "    \"routing\": {\n",
+    "    },\n",
+    "    \"requestPath\": {\n",
+    "      \"classifier\": \"seldonio/mock_classifier_rest:1.3\"\n",
+    "    },\n",
+    "    \"metrics\": []\n",
+    "  },\n",
+    "  \"data\": {\n",
+    "    \"names\": [\"proba\"],\n",
+    "    \"ndarray\": [[0.43782349911420193]]\n",
+    "  }\n",
+    "}\n",
+    "```"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.9"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
Always clean up the resources at the end.

Reply:
Addressed